src/components/interface/EcosystemHeader/EcosystemHeader.tsx

```diff
@@ -0,0 +1,75 @@
+import { SystemStyleObject, Theme, Box } from "@mui/system";
+import { Panel, Typography } from "src/components/atomic";
+
+export type EcosystemHeaderProps = {
+    lpOptimizerTag: string;
+    lpOptimizerCount: number;
+    alphaVaultTag: string;
+    alphaVaultCount: number;
+};
+
+const EcosystemHeader = ({lpOptimizerTag, lpOptimizerCount, alphaVaultTag, alphaVaultCount}: EcosystemHeaderProps) => {
+  
+    const titleStyles: SystemStyleObject<Theme> = { 
+        fontSize: '40px', 
+        lineHeight: '1.2', 
+        color: '#E5E1F9',
+        fontWeight: '700',
+    };
+
+    const copyStyles: SystemStyleObject<Theme> = {
+        fontSize: '14px',
+        color: '#9B97AD', 
+        marginTop: "8px",
+    }
+    
+    const tagStyles: SystemStyleObject<Theme> = {
+        fontSize: '14px',
+    };
+
+    const boxStyles: SystemStyleObject<Theme> = {
+        display: "flex",
+        borderRadius: "4px",
+        padding: "0px 8px",
+    }
```

Comment:
Move all the styles into ./styles.ts file nested in the same dir as component. Let's avoid recreating constants on every render.

Reply:
Resolved. https://github.com/Voltz-Protocol/voltz-ui/pull/295/commits/a44c3c48e2a2660e1fc14f6eeeda646152d2edea